fdbserver/DDTeamCollection.actor.cpp

```diff
@@ -48,6 +48,32 @@ FDB_DEFINE_BOOLEAN_PARAM(ForReadBalance);
 FDB_DEFINE_BOOLEAN_PARAM(PreferLowerReadUtil);
 FDB_DEFINE_BOOLEAN_PARAM(FindTeamByServers);
 
+namespace data_distribution {
+int EligibilityCounter::fromGetTeamRequest(GetTeamRequest const& req) {
+	// equivalent to bit set operation
+	return req.preferLowerDiskUtil * EligibilityCounter::LOW_DISK_UTIL +
+	       req.preferLowerReadUtil * EligibilityCounter::LOW_CPU;
```

Comment:
The `GetTeamRequest::preferLowerReadUtil` stands for lower CPU and lower ReadUtil right now. Are they equivalent? I am assuming that if we don't want to add a new variable to GetTeamRequest(e.g. `GetTeamRequest::preferLowerCPU`), it might help if we can comment `CPU` stat is for admittance to eligible pool, and `ReadLoad` is for sorting inside the pool.